src/extractors/gitmojiextractor.py

```diff
@@ -0,0 +1,56 @@
+"""
+Copied and changed from nerdextractor.py in https://github.com/fdw/rofimoji
+"""
+
+
+import html
+from pathlib import Path
+import json
```

Comment:
`json` is never used, so we can probably delete it.

Reply:
Makes sense